tapchannel/aux_funding_controller.go

```diff
@@ -1497,6 +1497,26 @@ func (f *FundingController) processFundingReq(fundingFlows fundingFlowIndex,
 		}
 	}()
 
+	// We need to limit the number of different fungible assets (asset IDs)
+	// we allow to be commited to a single channel. This is to make sure we
+	// have a decent number of HTLCs available. See Godoc of maxNumAssetIDs
+	// for more information.
+	//
+	// TODO(guggero): This following code is obviously wrong and needs to be
```

Comment:
Is this TODO still applicable?

Reply:
Yes, we need to actually count the number of different asset IDs (tranches) that are used for funding a channel, once we allow grouped assets to be used. So this will change in 0.6.